libvncclient/rfbproto.c

```diff
@@ -1585,6 +1591,52 @@ SendKeyEvent(rfbClient* client, uint32_t key, rfbBool down)
   return WriteToRFBServer(client, (char *)&ke, sz_rfbKeyEventMsg);
 }
 
+/*
+ * Resize client
+ */
+rfbBool
```

Comment:
Can this be made static?

Reply:
Tested and works also making this static